soroban-spec/src/gen/rust/types.rs

```diff
@@ -31,7 +31,7 @@ pub fn generate_struct(spec: &ScSpecUdtStructV0) -> TokenStream {
         });
         quote! {
             #[soroban_sdk::contracttype(export = false)]
-            #[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd)]
+            #[derive(Debug, Clone, Copy, Eq, PartialEq, Ord, PartialOrd)]
```

Comment:
Unfortunately we can't add `Copy` to struct or enum types that can contain other types. `Copy` requires that all contained types are also `Copy`, and not all types a struct or enum can contain are `Copy`.

For example, `Vec` cannot be made `Copy`, and this means that any custom type `struct` that contains a `Vec` would fail to compile.